src/example_config.yaml

```diff
@@ -11,6 +11,8 @@ pali_concordance_filepath: "./sc-renumber-segments/data/all_pali_concordance.jso
 pali_canon_path: "~/ms_palicanon"
 reference_root_path: "./bilara-data/reference/pli/ms"
 
+exclude_dirs: ['name', 'xplayground']
```

Comment:
I added this here so that when developing locally, and you copy this file to the top level directory, the attribute is present.